editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -1,12 +1,12 @@
 created: 20131129090249275
-modified: 20200507203622933
+modified: 20211111163520352
 tags: [[Working with TiddlyWiki]]
 title: GettingStarted
 type: text/vnd.tiddlywiki
 
-Click here to download an empty copy of TiddlyWiki: {{$:/editions/tw5.com/snippets/download-empty-button}}
+In its basic form, TiddlyWiki is an HTML file. Click here to download an empty HTML copy of TiddlyWiki: {{$:/editions/tw5.com/snippets/download-empty-button}}
 
-The next step is to choose a method for saving changes. There's a wide variety of methods available, with different features and limitations. Click on the badge for a method to see more information about it. You can also click on one of the platform filters to restrict the listing to methods that work with that platform.
+Because it is an HTML file, there is no built-in mechanism for saving changes except the [[fallback saver|Saving with the HTML5 fallback saver]], which can be somewhat inconvenient. But there are other methods for saving. So the next step is to choose a method for saving changes. There's a wide variety of methods available, with different features and limitations. Click on the badge for a method to see more information about it. You can also click on one of the platform filters to restrict the listing to methods that work with that platform.
```

Comment:
Instead of "fallback saver" we should write "browser default file saver" ... which is part of every browser and users should be used to it, if the download a file from the internet. ... 

That we use it as a "fallback saver" is just a coincidence.

Reply:
I chose that name because that's what it's called in the tiddler that defines how it is used. Why introduce confusion here?

Reply:
Browser default file saver might mislead people to try to save from the file menu.

Reply:
Then we should change the description of the "fallback saver" in general. Using a "fallback" has a negative meaning. Whereas it is the standard in every browser. So it is the "standard saver" and _not_ a "fallback".

Reply:
There are currently [5 tiddlers which IMO use the negative wording](https://tiddlywiki.com/#GettingStarted%20-%20Safari:%5B%5BGettingStarted%20-%20Safari%5D%5D%20%5B%5BGettingStarted%20-%20Chrome%5D%5D%20%5B%5B%22savetiddlers%22%20Extension%20for%20Chrome%20and%20Firefox%20by%20buggyj%5D%5D%20%5B%5B%22file-backups%22%20Extension%20for%20Firefox%20by%20pmario%5D%5D%20%5B%5BSaving%20with%20the%20HTML5%20fallback%20saver%5D%5D), which should be changed.

Reply:
I've changed the naming in the 2nd commit.